warehouse/slave/worker.go

```diff
@@ -531,7 +534,7 @@ func (w *worker) processStagingFile(ctx context.Context, job *payload) ([]upload
 	if err := w.processSingleStagingFile(ctx, jr, &job.basePayload, stagingFileInfo{
 		ID:       job.StagingFileID,
 		Location: job.StagingFileLocation,
-	}); err != nil {
+	}, ""); err != nil {
```

Comment:
We are passing an empty string here because this code path is not being used anymore with upload_v2 ?

Reply:
I think we can pass the hash.